coach/src/concept-coach/base.cjsx

```diff
@@ -109,7 +110,11 @@ ConceptCoach = React.createClass
     navigator.off('show.*', @updateView)
 
   getAllowedView: (userInfo) ->
-    {defaultView} = @props
+    {defaultView, collectionUUID, enrollmentCode} = @props
+    ignoreDefaultView = @state?.ignoreDefaultView or false
+    course = User.getCourse(collectionUUID, enrollmentCode)
+
+    # console.info(userInfo, course)
     if not userInfo.isLoaded
```

Comment:
stray `console.log`; commented out tho, so no biggy but also no reason to leave it in